fingerprint_pro_server_api_sdk/extend_exception.py

```diff
@@ -0,0 +1,7 @@
+from fingerprint_pro_server_api_sdk.rest import KnownApiException
+
+
+def extend_exception(exception, error_object):
+    if exception.status == 429:
+        error_object.retry_after = int(exception.headers.get('retry-after'))
```

Comment:
Will it also work for edge-case in which `retry-after` would be missing in headers?